src/utils/searchAndSort.ts

```diff
@@ -61,9 +63,21 @@ const searchConditionMet = <T extends Record<string, unknown>>(result: T, condit
       return searchValues.some((value) => resultValue.includes(value));
     } else if (condition.type === 'Fuzzy') {
       return searchValues.some((value) => {
-        // Split the search value into trigrams and see if the result field contains any trigram
+        // Trigrams don't work with single letter searches
+        if (value.length === 1) {
```

Comment:
Seems like we may want to include up to 3 length for this case. For reference the console has 25 deliverables under Sampson Road. If I search `sa`, or `sam`. I get 8 of the 25 on the former and no results on the latter. works great for 4 or more!

Reply:
This is fixed in the latest commit